mobile/src/main/java/org/openhab/habdroid/ui/OpenHABAboutActivity.java

```diff
@@ -61,81 +80,243 @@ public void finish() {
+                    .subText(R.string.about_license)
+                    .icon(R.drawable.ic_account_balance_grey_24dp)
+                    .setOnClickAction(MaterialAboutItemOnClickRedirect(URL_TO_GITHUB + "/blob/master/LICENSE"))
+                    .build());
+            appCard.addItem(new MaterialAboutActionItem.Builder()
+                    .text(R.string.title_activity_libraries)
+                    .icon(R.drawable.ic_developer_mode_grey_24dp)
+                    .setOnClickAction(() -> new LibsBuilder()
+                            .withActivityTheme(getTheme())
+                            .withFields(R.string.class.getFields())
+                            .withLicenseShown(true)
+                            .withAutoDetect(true)
+                            .start(context))
+                    .build());
 
-            // don't recreate the fragments when changing tabs
-            viewPager.setOffscreenPageLimit(1);
+            MaterialAboutCard.Builder ohServerCard = new MaterialAboutCard.Builder();
+            ohServerCard.title(R.string.about_server);
+            if(conn == null || mOpenHABVersion == 0) {
+                ohServerCard.addItem(new MaterialAboutActionItem.Builder()
+                        .text(R.string.error_about_no_conn)
+                        .icon(R.drawable.ic_info_outline)
+                        .build());
+            } else {
+                String apiVersion = getApiVersion(conn);
+                if(TextUtils.isEmpty(apiVersion)) {
+                    apiVersion = getString(R.string.unknown);
+                }
+                ohServerCard.addItem(new MaterialAboutActionItem.Builder()
+                        .text(R.string.info_openhab_apiversion_label)
+                        .subText(apiVersion)
+                        .icon(R.drawable.ic_info_outline)
+                        .build());
 
-            mPagerAdapter = new AboutPagerAdapter(getChildFragmentManager(),
-                    getActivity(), getArguments());
-            viewPager.setAdapter(mPagerAdapter);
+                String uuid = getServerUuid(conn);
+                if(TextUtils.isEmpty(uuid)) {
+                    uuid = getString(R.string.unknown);
+                }
+                ohServerCard.addItem(new MaterialAboutActionItem.Builder()
+                        .text(R.string.info_openhab_uuid_label)
+                        .subText(uuid)
+                        .icon(R.drawable.ic_info_outline)
+                        .build());
 
-            final TabLayout tabLayout = (TabLayout) view.findViewById(R.id.tab_layout);
-            tabLayout.setupWithViewPager(viewPager);
-            tabLayout.setTabsFromPagerAdapter(mPagerAdapter);
-            tabLayout.setTabMode(TabLayout.MODE_FIXED);
+                if (mOpenHABVersion == 1) {
+                    String secret = getServerSecret(conn);
+                    if (!TextUtils.isEmpty(secret)) {
```

Comment:
That negation is wrong.

Maybe even hide the card item completely in that case, given secret is only populated if the cloud connector is installed?